configs/bundle_config.yaml

```diff
@@ -141,6 +141,20 @@ databundles:
     - data/hydrobasins/hybas_world_lev04_v1c.shp
     - data/hydrobasins/hybas_world_lev05_v1c.shp
 
+  # global data for hydrobasins
+  bundle_hydrobasins:
+    countries: [Earth]
+    tutorial: false
+    category: common
+    destination: "data/hydrobasins"
+    urls:
+      hydrobasins:
+        base_url: https://data.hydrosheds.org/file/HydroBASINS/standard/
+        suffixes: ["af", "ar", "as", "au", "eu", "gr", "na", "sa", "si"]
+    unzip: true
+    output:
+    - data/hydrobasins/*.shp
+
```

Comment:
As comments:
- the output file could be hybas_world_v1c.shp, that represent the merged file, value not to be loaded from the config, but from here
- we could add a databundle for the tutorial, which is the same as this one, but limited to africa. So the changes would be: tutorial=true, countries=["Africa"], and suffixes=["af"]

Pretty cool! :D
Once this works, we can think of creating the databundles for the tutorials and plan to go for zenodo.
We can try again zenodo sandbox and when it works we can move to the default